man/man5/zfs-module-parameters.5

```diff
@@ -1658,6 +1658,25 @@ as a percentage of \fBzfs_dirty_data_max\fR.  This should be less than
 Default value: \fB20\fR% of \fBzfs_dirty_data_max\fR.
 .RE
 
+.sp
+.ne 2
+.na
+\fBzfs_fallocate_reserve_percent\fR (uint)
+.ad
+.RS 12n
+Since ZFS is a copy-on-write filesystem with snapshots, blocks cannot be
+preallocated for a file in order to guarantee that later writes will not
+run out of space.  Instead, fallocate() space preallocation only checks
+that sufficient space is currently available in the pool or the user's
+project quota allocation, and then creates a sparse file of the requested
+size. The requested space is inflated by \fBzfs_fallocate_reserve_percent\fR
```

Comment:
I would interpret "inflated by" to mean "increased by", i.e. you need requested_size + requested_size * zfs_fallocate_reserve_percent`.  Suggest rewording this:

```suggestion
size. The requested space is multiplied by \fBzfs_fallocate_reserve_percent\fR
```

What happens if this is set between 1 and 99?  We check for less space than was requested?  That might be reasonable if we expect compression (or dedup) to take effect.

Reply:
Yes, I mentioned this in the earlier discussion "In theory it would even be possible to set zpl_fallocate_ratio < 100 to account for compression."  Whether that is sane or not depends on the usage (i.e. it would be bad to assume compression if receiving .mpg files), but at least it is conceivable that this makes sense for some workloads, and would have to actively be changed by the administrator, so I don't think it should be prevented.

Reply:
I didn't realize that this would result in a separate patch rather than being merged into the original commit.  It doesn't make sense IMHO to have a separate one-line patch for this tiny correction, so I've re-pushed with this change merged in.